docker/cloudbuild.yaml

```diff
@@ -107,6 +106,27 @@ steps:
     docker push getsentry/sentry:$COMMIT_SHA
     docker tag us.gcr.io/$PROJECT_ID/sentry:$COMMIT_SHA getsentry/sentry:latest
     docker push getsentry/sentry:latest
+- name: 'gcr.io/cloud-builders/docker'
+  id: docker-push-py3
+  waitFor:
+    - e2e-test-py3
+  secretEnv: ['DOCKER_PASSWORD']
+  entrypoint: 'bash'
+  args:
+  - '-e'
+  - '-c'
+  - |
+    # Only push to Docker Hub from master
+    [ "$BRANCH_NAME" != "master" ] && exit 0
+    # Need to pull the image first due to Kaniko
+    docker pull us.gcr.io/$PROJECT_ID/sentry:$COMMIT_SHA-py3
+    echo "$$DOCKER_PASSWORD" | docker login --username=sentrybuilder --password-stdin
+    docker tag us.gcr.io/$PROJECT_ID/sentry:$COMMIT_SHA getsentry/sentry:$SHORT_SHA-py3
```

Comment:
Doesn't $COMMIT_SHA need a py3 suffix here?

Reply:
Nevermind, I see why.

Reply:
it does 🤦🏻

Reply:
Then I clearly am not understanding how any of this works, haha

Reply:
It's essentially a `mv` command. `docker tag <current_on_gcb> <what_we_want_on_docker_hub>`. Both need to have `-py3` as that's how we build the Py3 variant in earlier steps.